src/components/Search.vue

```diff
@@ -16,11 +16,48 @@
     <a @click="clearInput">
       <Icon v-if="modelValue" name="close" size="12" class="mb-1" />
     </a>
+    <UiDropdown
+      class="search-dropdown"
+      @redirect="redirectSearch"
+      :items="searchOptions"
+      v-if="!modal"
+    >
+      <span v-text="searchSelectedOption" />
+      <i class="iconfont iconarrow-down ml-1"></i>
+    </UiDropdown>
   </div>
 </template>
 
 <script>
 export default {
+  data: function() {
+    return {
+      searchOptions: [
+        { text: 'Spaces', action: 'redirect', options: { route: 'home' } },
```

Comment:
You should use translation key for the text and make sure it's translated on the dropdown component.